recipes/conda_gitenv/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: conda-gitenv
```

Comment:
@jakirkham Apologies, will do. 

Interesting. Would it be easy to have a simple automated check on the staged-recipes PR to detect this type of error earlier, before failing post merge?

I'll push a fresh PR with the correction, thanks.